specification/resources/resource-manager/readme.md

```diff
@@ -57,19 +57,10 @@ tag: package-managedapplications-2018-06
 ```
 
 ``` yaml $(package-deploymentscripts)
-tag: package-2019-11
+tag: package-deploymentscripts-2019-10-preview
```

Comment:
Revert the change made in https://github.com/Azure/azure-rest-api-specs/pull/7839/files/8668e09bf93fd5a9cbaf00980a6cf50cdb6dee33#diff-d474de52c790d7028fb81d4abf0b9618R60, so that `$(package-deploymentscripts)` matches `tag: package-deploymentscripts-2019-10-preview`.